config/studio/site-policy-config.xml

```diff
@@ -16,7 +16,7 @@
   -->
 <site-policy>
     <statement>
-        <target-path-pattern>/static-assets/.*</target-path-pattern>
+        <target-path-pattern>/(?!scripts/.*|templates/.*|static-assets/app/.*).*</target-path-pattern>
```

Comment:
This is the only change we need (along with the version number update). The rest can be removed from the PR. UM will not change this anymore, so we're good. The reason for not accepting the rest is that we expect this BP to work with older versions of Crafter.

Reply:
Updated

Reply:
@sumerjabri I updated this. This BP (and other BPs to be updated) have the supported version from 4.1.0 to the latest.
If we update the Studio version to 4.1.2, then the following pipeline from 4.0.0.1 to 4.1.2 won't be applied even if we create a site from this BP using later versions: https://github.com/craftercms/studio/blob/support/4.1.x/src/main/resources/crafter/studio/upgrade/pipelines.yaml#L686
Just to make sure are we ok with this?